ui/.env.development

```
@@ -1,3 +1,5 @@
 NEXT_PUBLIC_GQL_ENDPOINT='http://localhost:${SERVER_PORT:-4000}/graphql'
 NEXT_PUBLIC_SOCKET_ENDPOINT='ws://localhost:${SERVER_PORT:-4000}/socket'
-NEXT_PUBLIC_S3_BUCKET='dev-streaming-orcasound-net'
\ No newline at end of file
+NEXT_PUBLIC_S3_BUCKET='dev-streaming-orcasound-net'
+NEXT_PUBLIC_SHIPNOISE_API_URL='/api/shipnoise'
+SHIPNOISE_API_URL='https://orca-shipnoise-sjdtow.fly.dev'
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)